src/main/java/heronarts/lx/LXComponent.java

```diff
@@ -884,11 +885,19 @@ protected LXComponent addParameter(String path, LXParameter parameter) {
     }
     if (parameter instanceof ColorParameter) {
       // NOTE: order is important, brightness must be saved/loaded first
+      // TODO: Why?
```

Comment:
When any of the HSB params changes, the color is updated, and vice versa. But there's potentially more information in the HSB params than the 32-bit color. Particularly, e.g. if brightness or saturation is 0, then there's no hue information to extract from the 32-bit color value. The position of the color slider is lost.

So HUE is restored LAST to stop the later restoration of a potentially-0 brightness/saturation value from clobbering it.

I am not sure this is *strictly* necessary anymore, I may have fixed it another way (and I aim to make this cleaner with AggregateParameter).